include/zephyr/posix/time.h

```diff
@@ -83,7 +83,7 @@ extern "C" {
 
 static inline int32_t _ts_to_ms(const struct timespec *to)
 {
-	return (to->tv_sec * MSEC_PER_SEC) + (to->tv_nsec / NSEC_PER_MSEC);
+  return (int32_t)((to->tv_sec * MSEC_PER_SEC) + (to->tv_nsec / NSEC_PER_MSEC));
```

Comment:
Maybe it's rendering incorrectly on mobile, but the indent does not look like a tab

Reply:
Fixed